### ISSUE TITLE
Improve pluralization of model classes ending in "s"

### ISSUE THREAD

**Issue by phenning** [dotnet/Scaffolding#2376]
Create a model called "Class" or similar.

Add a new scaffolded item - API with read/write endpoints

Expected: 
```
        group.MapGet("/", () =>
        {
            return new [] { new Class() };
        })
        .WithName("GetAllClasses")
        .WithOpenApi();
```

Actual:
```
        group.MapGet("/", () =>
        {
            return new [] { new Class() };
        })
        .WithName("GetAllClasss")
        .WithOpenApi();
```

**Comment by deepchoudhery** [comment]
yup I believe this [PluralizationService](https://learn.microsoft.com/en-us/dotnet/api/system.data.entity.design.pluralizationservices.pluralizationservice?view=netframework-4.8)should be good

**Comment by deepchoudhery** [comment]
fixed with 6.0.14, 7.0.7, and 8.0 Preview 5. Servicing releases next week using Humanizer. Closing.
